ui/webui/src/components/storage/DiskEncryption.jsx

```diff
@@ -154,6 +155,15 @@ const PasswordFormFields = ({
                         >
                             {_("Must be at least 8 characters")}
                         </HelperTextItem>
+                        {ruleAscii &&
+                        <HelperTextItem
+                          id={idPrefix + "-password-rule-ascii"}
+                          isDynamic
+                          variant="warning"
+                          component="li"
+                        >
+                            {_("The passphrase you have provided contains non-ASCII characters. You may not be able to switch between keyboard layouts when typing it.")}
```

Comment:
I would change the formulation a bit, as both sentences look kinda disjoint. Maybe something like:

"The passphrase you have provided contains non-ASCII characters, which you might not be able to enter, as keyboard switching if often not available when LUKS passphrase is requested."

Maybe @VladimirSlavik might also have some ideas. :)

Reply:
The message is copied from `pyanaconda/core/constants.py` to match the warning you get in the GTK UI. If you have some suggestion, we should change it everywhere.